src/SFML/Window/OSX/SFWindowController.mm

```diff
@@ -471,18 +471,9 @@ -(void)resizeTo:(unsigned int)width by:(unsigned int)height
         // Add titlebar height.
         height += static_cast<unsigned int>([self titlebarHeight]);
 
-        // Corner case: don't set the window height bigger than the screen height
-        // or the view will be resized _later_ without generating a resize event.
```

Comment:
Maybe this comment was there for a reason and we have to handle the corner case described here.
I will be able to check on a macbook this week.

Reply:
If it's resized later without generating an event that doesn't matter because we just emitted the event.

Reply:
Maybe the view will be resized to a different size, one that fits in screen height, without notifying the user.